index.html

```diff
@@ -0,0 +1,222 @@
+<!DOCTYPE html>
+<html lang="pl">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script src="https://kit.fontawesome.com/398c36776a.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header class="header">
+        <div class="header_hero_bg">
+            <nav class="header_nav container">
+
+                <a href="#default" class="logo"></a>
+                <ul class="header_right">
+                    <a href="#team">The Team</a>
+                    <a href="#pricing ">Pricing </a>
+                    <a href="#features">Features</a>
+                </ul>
+            </nav>
+
+            <div class="container header-box ">
+                <div class="header-box-left">
+                    <h1>Beautiful Free Nova template </h1>
+                    <p>A top notch premium quality template for your next web project.</p>
+                    <button class="header-box-left-button"">Download FREE!!!</button>
+                </div>
+            <div class=" header-box-right ">
+                        <div class=" header-box-right-bg">
```

Comment:
Zabrakło obrócenia - ten element też bym zrobił jako `img` bo to nie tło, a faktycznie jakiś obraz do pokazania użytkownikowi. Brakuje też zaokrąglenia tj. `border-radius`

Do obrotu można użyć `rotate`: https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotate